test/specs/elements/Divider/Divider-test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import Divider from 'src/elements/Divider/Divider'
+import * as common from 'test/specs/commonTests'
+
+describe('Divider', () => {
+  common.isConformant(Divider)
```

Comment:
There are also common tests for the classNames, reference https://github.com/TechnologyAdvice/stardust/blob/master/test/specs/elements/Label/Label-test.js